### DISCUSSION TITLE
Suggestion about unethical recommended suggestions.

### DISCUSSION THREAD

**Discussion by requireScript** [pieces-app/support#140] [Ideas]
Whenever an user is trying to generate unethical stuff, Pieces still gives a recommended question after Pieces rejects assistance with the unethical question.
![image](https://github.com/pieces-app/support/assets/164247124/14e397e4-6ad7-4371-a671-6368ae9d7796)

**Comment by sam-at-axiotic** [comment]
Thanks for highlighting this @requireScript, good catch. Removing suggestions for unethical messages is something we are currently planning.
